rules/helm/package.tmpl.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+build_dir="tmp/build/${PACKAGE_DIR}"
+mkdir -p "${build_dir}"
+
+cp -L -R "${PACKAGE_DIR}"/* "${build_dir}"
+
+# Generated files ( here TARS, GENERATED ) are not part of the source code
+# to be able to use them, we have to copy them
+for t in ${TARS}; do
+  tar xf "${t}" -C "${build_dir}"  > /dev/null
+done
+
+for g in ${GENERATED}; do
+  cp "${g}" "${build_dir}"/ > /dev/null
+done
+
+"${HELM}" init --client-only > /dev/null
+
+version='v0.0.0-{{ (ds "workspace_status").STABLE_GIT_COMMIT_SHORT }}'
+# If the branch name is a valid SemVer, use it as the chart version.
+if [[ -n "$(echo '{{ (ds "workspace_status").STABLE_GIT_BRANCH }}' | awk '/(?in)^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-([a-z-][\da-z-]+|[\da-z-]+[a-z-][\da-z-]*|0|[1-9]\d*)(\.([a-z-][\da-z-]+|[\da-z-]+[a-z-][\da-z-]*|0|[1-9]\d*))*)?(\+[\da-z-]+(\.[\da-z-]+)*)?$/ { print $0 }')" ]]; then
+  version='{{ (ds "workspace_status").STABLE_GIT_BRANCH }}'
```

Comment:
That looks… horrible? Scary? Nightmare-inducing? 😛 

Since the workspace status script is Ruby, can we just make _that_ give us a semver string instead, falling back to `v0.0.0-{{…}}`? Call it `STABLE_RELEASE_VERSION` or something?

Reply:
The script was rewritten using Ruby.